### ISSUE TITLE
CountVariants bogus error message when file missing and -L is used

### ISSUE THREAD

**Issue by akiezun** [broadinstitute/gatk#1896]
```
./gatk-launch CountVariants -V fred -L 20
```

blows up with a message 

```
A USER ERROR has occurred: We currently require a sequence dictionary (from a reference or source of reads) to process intervals. This restriction may be removed in the future.
```

which makes no sense because `fred` does not even exist

**Comment by droazen** [comment]
Well if it doesn't exist, then there's no sequence dictionary available, right? :)

**Comment by akiezun** [comment]
true. the question is though, will the restriction of providing an existing input file  `be removed in the future`?

**Comment by droazen** [comment]
Maybe it will -- having input files causes a lot of hassle/bugs! We'd be better off without them.

**Comment by akiezun** [comment]
it's not so much the input files themselves, maybe it's that we have to do something different for each of them.

**Comment by vdauwera** [comment]
The parser does seem to have questionable priorities in what to complain about first -- but I like the idea of doing away with input files. We could just put all the data inside GATK,  and require that users commit their data into the codebase in order to run analyses on it. That way the data would go through code review first which would catch all possible issues. /insanetrolllogic

**Comment by akiezun** [comment]
hmm. user data is problematic. Let's use our data and just output the same correct results for everyone, everytime.

**Comment by vdauwera** [comment]
It would be the best way to ensure reproducibility

**Comment by lbergelson** [comment]
Now that github doesn't charge us for large file storage this suggestion is totally reasonable.

**Comment by ldgauthier** [comment]
This is fixed in 4.1.4.1.
